aspnetcore/security/app-secrets/samples/3.x/UserSecrets/Program2.cs

```diff
@@ -0,0 +1,29 @@
+﻿#if never
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace UserSecrets
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            #region snippet_Host
+            var host = new HostBuilder()
+                .ConfigureAppConfiguration((hostContext, builder) =>
+                {
+                    // Add other providers for JSON, etc.
+
+                    if (hostContext.HostingEnvironment.IsDevelopment())
+                    {
+                        builder.AddUserSecrets<Program>();
+                    }
+                })
+                .Build();
+                #endregion
```

Comment:
This is the new stuff. I simplified from the snippet I pasted into the linked issue, to make it more consistent with [this approach](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-2.1#shutdown-timeout).